common/utils.go

```diff
@@ -0,0 +1,71 @@
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
```

Comment:
I personally see these methods under a different package, rather than common/utils.go. It seems to me that this specifically relates to our client implementation for sqs.

Reply:
I mean, I can put them inside an `sqs` folder, but there's barely 50 lines of code to move there. `common` to me seems fine, it is shared between source and connector. I'd personally rather have less packages than more.

Reply:
What do you think then @raulb ?

Reply:
@raulb can you please check these comments, I think this is an important PR to merge before updating our dependencies for the release